server/fakeData.js

```diff
@@ -0,0 +1,32 @@
+export const user1 = {
```

Comment:
delete this file and add your fake data in your server through GraphQL queries. Thank you.